### ISSUE TITLE
Extra slash included in URL paths

### ISSUE THREAD

**Issue by lily-mara** [Joedmin/xExtension-readeck-button#8]
I'm using Readeck 0.15.3, Freshrss 1.24.3, and I'm trying to set up this extension. After creating my API token and pasting it in the extension settings, I found that attempting to connect failed with a HTTP 308. I looked in my Readeck logs and found that the request was sent to the path `/api//profile` instead of the expected `/api/profile`. I opened the button controller class and edited the URL fragment, changing it from `/profile` to `profile`.

https://github.com/Joedmin/xExtension-readeck-button/blob/bba60680e3c06cd64365e855acc258c616aac23a/Controllers/readeckButtonController.php#L46

After making this change, I was able to log in, but I could not save any items in Readeck as other actions are using the double slash, the POST request to `/api//bookmarks` is now also failing with a 308. It would be nice if Readeck's router filtered out the extra slashes but it seems that it rejects them.

```
time="2024-10-30T05:38:10Z" level=info msg=http @id=boxxie/VkKQuDNrGc-000442 elapsed_ms=0.07885 http_method=GET http_proto=HTTP/1.1 length=65 path=/api//profile remote_addr=192.168.50.5 status=308 ua=
time="2024-10-30T05:44:48Z" level=info msg=http @id=boxxie/VkKQuDNrGc-000445 elapsed_ms=0.968694 http_method=GET http_proto=HTTP/1.1 length=494 path=/api/profile remote_addr=192.168.50.5 status=200 ua=

time="2024-10-30T05:46:08Z" level=info msg=http @id=boxxie/VkKQuDNrGc-000451 elapsed_ms=0.051613 http_method=POST http_proto=HTTP/1.1 length=0 path=/api//bookmarks remote_addr=192.168.50.5 status=308 ua=
```

**Comment by Joedmin** [comment]
Hello, thank you for reporting this issue. I must say I did not notice this isse, it's working fine for me (i gues Apache somehow handles this). I'm releasing a new version with this fix.

**Comment by lily-mara** [comment]
Can confirm it's fixed in 0.6, thank you for the quick turnaround!
